engine/src/test/java/io/camunda/zeebe/engine/processing/variable/mapping/VariableMappingTransformerTest.java

```diff
@@ -106,6 +106,7 @@ public void shouldNotEscapeCharactersInStaticExpression() {
                 mapping("Hello\tWorld", "tab"),
                 mapping("Hello\nWorld", "newline"),
                 mapping("Hello\rWorld", "carriageReturn"),
+                mapping("\"My Name is \"\"Zeebe\"\", nice to meet you\"", "doubleQoutes"),
```

Comment:
:thinking: Why do we have two double quotes `\"\"Zeebe\"\"`?

Reply:
Double quotes made me crazy 🤣